repos/clicommandlineparser.proj

```diff
@@ -2,7 +2,9 @@
 <Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))/dir.props" />
   <PropertyGroup>
-    <BuildCommandArgs>--restore --pack --configuration $(Configuration)</BuildCommandArgs>
+    <!-- In Windows -restore is already passed by default by build.cmd to build.ps1 -->
+    <BuildCommandArgs Condition="'$(OS)' == 'Windows_NT'">-pack -configuration $(Configuration)</BuildCommandArgs>
+    <BuildCommandArgs Condition="'$(OS)' != 'Windows_NT'">--restore --pack --configuration $(Configuration)</BuildCommandArgs>
```

Comment:
I remember making a comment on this about factoring out `--` vs. `-` so `-pack -configuration $(Configuration)` can be defined in one place, but I forget what happened to that PR. (Same comment for the other repo proj.)

Reply:
Oh yeah you're right. Wes asked me to throw this changes in a separate PR since the other PR was scoped to update the core-repos only.  Do you want to factor out -- and - in dir.props or leave it local to the projects that use them?

Reply:
I'd say `repos/dir.props` (probably what you mean) is good, and most repos just won't use it. If it makes some arg line particularly long don't hesitate to split it into property concats for readability.